src/main/java/uk/gov/companieshouse/company/profile/model/CompanyProfileDocument.java

```diff
@@ -16,7 +16,7 @@ public class CompanyProfileDocument {
     private String id;
 
     @Field("has_mortgages")
-    public boolean hasMortgages;
+    public Boolean hasMortgages;
```

Comment:
I guess it makes sense for this to be null if it didn't exist in the first place, but from a business perspective surely a company only has or hasn't got a mortgage - interesting ticket!